mobile/native/migrations/2023-03-06-005404_orders/up.sql

```diff
@@ -6,5 +6,9 @@ CREATE TABLE IF NOT EXISTS orders (
     contract_symbol TEXT NOT NULL,
     direction TEXT NOT NULL,
     order_type TEXT NOT NULL,
-    STATUS TEXT NOT NULL
+    STATUS TEXT NOT NULL,
```

Comment:
❓unrelated question: why is this column caps when the others are not?

Reply:
I was wondering as well.. I think it is dprint :/

Reply:
Yes, it is dprint. I'll leave it as is as it doesn't matter much :)